testutil/src/main/java/org/spine3/test/NullToleranceTest.java

```diff
@@ -389,7 +398,11 @@ private Object getDefaultValue(Class<?> type) {
                 }
             }
 
-            checkState(result != null);
+            checkState(
+                    result != null,
+                    String.format(
+                            "Could not generate a default value for %s type. Please, use NullToleranceTest.Builder#addDefaultValue",
```

Comment:
> NullToleranceTest.Builder#addDefaultValue

This syntax is for Javadocs. I would say that a developer in such a situation needs a method name to call.
And I would add `()` at the end.